### DISCUSSION TITLE
什么是 DCE 5.0 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#826] [Announcements]
# 什么是 DCE 5.0 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/dce/what-is-dce/

**Comment by liyuerich** [comment]
右侧内容滑动的时候能否左侧进行锚定，或者能快速知道从哪里入口的？ 或者能有面包屑导航，现在我在哪里如果滑动后就会迷失，因为资料比较多。
